src/wallet/wallet2.cpp

```diff
@@ -11276,12 +11317,14 @@ rct::multisig_kLRki wallet2::get_multisig_composite_kLRki(size_t n, const crypto
   size_t n_signers_used = 1;
   for (const auto &p: m_transfers[n].m_multisig_info)
   {
-    if (p.m_signer == ignore)
+    if (ignore_set.find(p.m_signer) != ignore_set.end())
       continue;
+
     for (const auto &lr: p.m_LR)
     {
       if (used_L.find(lr.m_L) != used_L.end())
         continue;
+
```

Comment:
:)

Reply:
Can you remove the whitespace change while you're at it ? :)

Reply:
oh, sorry! of course!